src/engine/DnCManager.cpp

```diff
@@ -115,9 +115,10 @@ void DnCManager::solve()
     };
 
     unsigned timeoutInSeconds = Options::get()->getInt( Options::TIMEOUT );
-    unsigned long long timeoutInMicroSeconds = timeoutInSeconds * MICROSECONDS_IN_SECOND;
-    struct timespec startTime = TimeUtils::sampleMicro();
+    unsigned long long timeoutInMicroSeconds = ( unsigned long long ) timeoutInSeconds * ( unsigned long long ) MICROSECONDS_IN_SECOND;
```

Comment:
No spaces for casts:
( unsigned long long ) timeoutInSeconds --> (unsigned long long)timeoutInSeconds